src/discussions/comments/data/slices.js

```diff
@@ -41,7 +42,7 @@ const commentsSlice = createSlice({
         totalPages: payload.pagination.numPages,
         hasMorePages: Boolean(payload.pagination.next),
       };
-      state.commentsInComments = { ...state.commentsInComments, ...payload.commentsInComments };
+      // state.commentsInComments = { ...state.commentsInComments, ...payload.commentsInComments };
```

Comment:
I guess this can just be removed now?

Reply:
You're right, I will remove it.
Thank you for pointing it out.